addon/utils/object-transforms.ts

```diff
@@ -0,0 +1,56 @@
+import { isPresent } from '@ember/utils';
```

Comment:
I'm finding the term "options" to be confusing throughout this file. Perhaps "object" would be more clear?

Reply:
These functions are all used in the context of config `options` [which is a term that is already used throughout the addon](https://github.com/adopted-ember-addons/ember-metrics/blob/fea9d4110c68ed9e43af5565ae630af6c6078de0/addon/metrics-adapters/intercom.js#L38). The naming here and the shape of `Record<string, string>` for the type was intended to reflect that. These util functions technically could be used for more general, non-ember-metrics objects, but they're internal API and it seemed consistent to me to limit the scope to the `options` concept here.

Reply:
That works! It might be prudent to move these to a `-private` directory to these to ensure it's really clear these are for our use only (i.e. `/addon/-private/utils`)?

Reply:
Sounds good to me! "Public API" tends to be just a suggestion in most cases anyway lol, might as well make it really explicit. Will break this out into a `/-private` folder.